examples/starter-webpack/webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   entry: path.resolve(__dirname, "src/index.tsx"),
   output: {
     path: path.resolve(__dirname, "dist"),
-    filename: "[name]-[hash].js",
+    filename: "[name]-[contenthash].js",
```

Comment:
`[hash]` is deprecated in prints a `DEP_WEBPACK_TEMPLATE_PATH_PLUGIN_REPLACE_PATH_VARIABLES_HASH` warning.